src/components/OneSlide/OneSlide.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import './OneSlide.css';
+
+const OneSlide = ({image}) => {
+
+  return (
+    <div className='one-slide'>
+      <img className='one-slide__image' src={image} alt='something'/>
```

Comment:
very descriptive alt tag 😂

with the fetch from the api, I would assume that the alt tag could be set to something like `${city.title}` if we destructured the title from the prop as well?